docs/source/github.md

```diff
@@ -34,6 +34,9 @@ We currently use the following fields:
 - `name` is the full name GitHub knows the user by.
 - `email` is the publicly visible email address (if any) for the user.
 - `access_token` is the token used to authenticate to GitHub.
+- `teams` is list of teams the user is part of, fetched only if
+  `fetch_teams` option is set to `True` on `GitHubOAuthenticator`.
+  The `read:orgs` scope is also required for this to work.
```

Comment:
```suggestion
  The `read:org` scope is also required for this to work.
```

Reply:
Can a user grant some but not all of these permissions, if so maybe we should have a fallback on just partially granted scopes by the user or handle an unauthorized response without crashing as we would do now.

Reply:
Can I do that in a follow-up?

Reply:
Sure!